From e59471616bef143c656c22d888f707c6819893c8 Mon Sep 17 00:00:00 2001
From: Pekka Paalanen <pekka.paalanen@collabora.com>
Date: Thu, 4 Apr 2019 15:54:07 +0300
Subject: [PATCH 11/45] xwayland: do not include weston.h

weston.h is a Weston frontend header, while this is a libweston plugin. A
libweston plugin cannot depend on Weston. Luckily the header is not actually
needed.

Signed-off-by: Pekka Paalanen <pekka.paalanen@collabora.com>
---
 xwayland/launcher.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xwayland/launcher.c b/xwayland/launcher.c
index 44083111..2c7da113 100644
--- a/xwayland/launcher.c
+++ b/xwayland/launcher.c
@@ -40,7 +40,6 @@
 #include <libweston/xwayland-api.h>
 #include "shared/helpers.h"
 #include "shared/string-helpers.h"
-#include "compositor/weston.h"
 
 static int
 weston_xserver_handle_event(int listen_fd, uint32_t mask, void *data)
-- 
2.21.0

